README.md

```diff
@@ -186,6 +186,88 @@ You can also use your own implementation of the `Spiral\RoadRunner\Http\PSR7Work
 $runner = $runner->withPsr7Worker($psr7Worker);
 ```
 
+## Temporal
+
+Temporal is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous long-running business logic in a scalable and resilient way.
+
+Explore more about Temporal on the official website https://temporal.io and in the sdk repository: https://github.com/temporalio/sdk-php.
+
+> If you want to add support for Temporal you need to install the SDK and configure workflows and activities.
```

Comment:
Is it useful info? Would it be described later in the README, or should I do it before? If so, where is that described? Maybe there should be a link?

Reply:
```suggestion
> If you want to add support for Temporal you need to install the SDK and configure workflows and activities as described below.
```